test/Validator/Text.php

```diff
@@ -78,8 +78,19 @@
                 (new Text)
                     ->email()
                     ->validate('test.com')
-                    ->errors()
-            )->toBe(['not an email']);
+                    ->errors()[0]
+                    ->__toString()
+            )->toBe('not an email');
+        });
+
+        it('can be translated', function () {
+            expect(
+                (new Text)
+                    ->email()
+                    ->validate('test.com')
+                    ->errors()[0]
+                    ->translate('hello')
```

Comment:
Does this require you to know what the first encountered error is going to be? e.g. if you have `->email()` and `->min(10)`, do you then need some extra logic to determine whether or not the value were an email address in order to know what you're translating?